scikeras/wrappers.py

```diff
@@ -1611,7 +1611,7 @@ def target_encoder(self):
     @register_keras_serializable()
     def r_squared(y_true, y_pred):
         """A simple Keras implementation of R^2 that can be used as a Keras
-        loss function.
+        metric function.
```

Comment:
Are lower values better for this metric?

Reply:
Good point. How about adding:
> Larger values indicate a better fit, with 1.0 representing a perfect fit.